v1/metrics.yaml

```diff
@@ -1429,7 +1429,7 @@ paths:
           name: ratelimit_route
           options:
             limits:
-              internal: 100
+              internal: 25
               external: 100
```

Comment:
Shouldn't the external limit be updated here as well?

Reply:
yes!